Solutions/Marain.Workflow.Api.MessageProcessingHost/Marain/Workflows/Api/MessageProcessingHost/Startup.cs

```diff
@@ -38,13 +38,13 @@ public void Configure(IWebJobsBuilder builder)
             var uri = new Uri(root["Operations:ControlServiceBaseUrl"]);
             services.AddOperationsControlClient(uri);
 
-            services.AddTenantedWorkflowEngineClient(sp =>
+            services.AddMarainWorkflowEngineClient(sp =>
             {
                 IConfiguration config = sp.GetRequiredService<IConfiguration>();
 
-                return new WorkflowEngineClientConfiguration
+                return new MarainWorkflowEngineOptions
```

Comment:
If you changed the setting name to be `Workflow:Client:BaseUrl` you should then be able to use `return config.Get<MarainWorkflowEngineOptions>();`